toolkit/scripts/toolchain/container/toolchain_build_in_chroot.sh

```diff
@@ -792,7 +792,7 @@ chmod -v 755 /usr/lib/libpython3.9.so.1.0
 chmod -v 755 /usr/lib/libpython3.so
 ln -sfv pip3.9 /usr/bin/pip3
 popd
-rm -rf Python-3.9.13
+rm -rf Python-3.9.14
```

Comment:
Revert unnecessarily changes to stage 1 toolchain.
```suggestion
rm -rf Python-3.9.13
```